src/plugins/privacy/activitystream/activitystream.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<extension version="3.9" type="plugin" group="privacy" method="upgrade">
+	<name>plg_privacy_activitystream</name>
+	<version>__DEPLOY_VERSION__</version>
+	<creationDate>5th October 2018</creationDate>
```

Comment:
@Mayaank12 
Use 
__RELEASE_DATE__
for creation date

Reply:
Done